src/scenes/AcceleratorRouter/Scoring/PhaseScores/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useMemo } from 'react';
+import { useRouteMatch } from 'react-router-dom';
+
+import { Box, Grid, Typography, useTheme } from '@mui/material';
+
+import { APP_PATHS } from 'src/constants';
+import { useLocalization } from 'src/providers';
+import strings from 'src/strings';
+import { PhaseScores as PhaseScoresType, Score, ScoreValue, getPhase } from 'src/types/Score';
+import { getLongDate } from 'src/utils/dateFormatter';
+
+import ScoreEntry from './ScoreEntry';
+import ScoreTotal from './ScoreTotal';
+
+interface ScorecardProps {
+  editable?: boolean;
+  onChangeValue?: (score: Score, value: ScoreValue) => void;
+  onChangeQualitative?: (score: Score, value: string) => void;
+  phaseScores?: PhaseScoresType;
+}
+
+const PhaseScores = ({ editable, onChangeValue, onChangeQualitative, phaseScores }: ScorecardProps) => {
+  const theme = useTheme();
+  const { activeLocale } = useLocalization();
+  const isEditing = useRouteMatch(APP_PATHS.ACCELERATOR_SCORING_EDIT);
+
+  const modifiedTime = useMemo(
+    () =>
+      phaseScores?.scores
+        .map((score: Score) => score.modifiedTime)
+        .sort()
+        .shift(),
+    [phaseScores]
+  );
+
+  if (!phaseScores) {
+    return null;
+  }
+
+  return (
+    <>
+      <Box
+        borderRadius={theme.spacing(1)}
+        padding={3}
+        margin={0}
+        sx={{
+          textAlign: 'center',
+          backgroundColor: theme.palette.TwClrBaseGray050,
+        }}
+      >
+        <Typography>{getPhase(phaseScores.phase)}</Typography>
+
+        <ScoreTotal isEditing={!!isEditing} phaseScores={phaseScores} />
+
```

Comment:
I think in the mocks, we will want to add scoring completed/not completed text here. 

https://www.figma.com/file/Ec5QnRsnzqEKeifzwPKR3k?node-id=1654:26115&mode=design#739587045

Reply:
Oops sorry followed the wrong prop! I see that this is part of the TotalScore!